src/ray/raylet/agent_manager.cc

```diff
@@ -97,5 +99,81 @@ void AgentManager::StartAgent() {
   monitor_thread.detach();
 }
 
+void AgentManager::CreateRuntimeEnv(const std::string &serialized_runtime_env,
+                                    CreateRuntimeEnvCallback callback) {
+  if (runtime_env_agent_client_ == nullptr) {
+    RAY_LOG(INFO) << "No available agent connection. Retry to create runtime env "
+                  << serialized_runtime_env << " later.";
```

Comment:
1) `serialized_runtime_env` is binary data? Is this human readable?
2) This may be clearer: `Runtime env agent is not registered yet. Will retry CreateRuntimeEnv later.`

Reply:
`serialized_runtime_env` is a readable json string now.